custom/_legacy/mvp/reports/va.py

```diff
@@ -0,0 +1,298 @@
+import datetime
+from couchdbkit import ResourceNotFound
+import dateutil
+from django.utils.safestring import mark_safe
+import logging
+import numpy
+import pytz
+from corehq.apps.indicators.models import DynamicIndicatorDefinition, CombinedCouchViewIndicatorDefinition
+from dimagi.utils.decorators.memoized import memoized
+from mvp.models import MVP
+from mvp.reports import MVPIndicatorReport
+
+
+class VerbalAutopsyReport(MVPIndicatorReport):
+    slug = "verbal_autopsy"
+    name = "Verbal Autopsy Report"
+    report_template_path = "mvp/reports/verbal_autopsy.html"
+    flush_layout = True
+    hide_filters = True
+    fields = ['corehq.apps.reports.filters.users.UserTypeFilter',
+              'corehq.apps.reports.filters.select.GroupFilter']
+    emailable = True
+
+    @property
+    def timezone(self):
+        return pytz.utc
+
+    @property
+    def num_prev(self):
+        try:
+            return int(self.request.GET.get('num_prev'))
+        except (ValueError, TypeError):
+            pass
+        return 12
+
+    @property
+    def current_month(self):
+        try:
+            return dateutil.parser.parse(self.request.GET.get('current_month'))
+        except (AttributeError, ValueError):
+            pass
+
+    @property
+    @memoized
+    def template_report(self):
+        if self.is_rendered_as_email:
+            self.report_template_path = "mvp/reports/verbal_autopsy_email.html"
+        return super(VerbalAutopsyReport, self).template_report
+
+    @property
+    def report_context(self):
+        report_matrix = []
+        month_headers = None
+        for category_group in self.indicator_slugs:
+            category_indicators = []
+            total_rowspan = 0
+            for slug in category_group['indicator_slugs']:
+                try:
+                    indicator = DynamicIndicatorDefinition.get_current(
+                        MVP.NAMESPACE, self.domain, slug,
+                        wrap_correctly=True,
+                        use_new_db=True,
```

Comment:
I'm referring to this https://github.com/SEL-Columbia/commcare-hq/blob/va/custom/_legacy/mvp/reports/__init__.py#L40

Because this:
https://github.com/SEL-Columbia/commcare-hq/blob/va/custom/_legacy/mvp/reports/__init__.py#L63